eos_data_distribution/ndn/command.py

```diff
@@ -47,6 +47,17 @@ def makeCommandInterest (*args, **kwargs):
     'flagbit': 'setFlagBit'
 }
 
+def addNextHop(faceUri, name, cost=0, *args, **kwargs):
+    face = Face(faceUri)
```

Comment:
This needs to be a GLibUnixFace in order for it to be integrated with the mainloop, right? I don't quite understand why we're connecting to somebody else's machine here...